### PR TITLE
Add vendor variable

### LINKED ISSUE #46: Add support for customizing the source name
Infrequently someone might want to customize the name of the lead source that LC sends to TrustedForm. Currently, it's hard coded to the name of the flow source that generated the lead. 

https://activeprospect.slack.com/archives/C034BK84K/p1515599655000159

### PR DESCRIPTION
**Pull request by chasingSublimity**
closes #46

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org)
 
+## [1.6.0] - 2018-01-10
+### Added
+- Add support for setting custom vendor name
```
Comment (jgrayson676) [inline CHANGELOG.md]: Minor, but could you add a link to the issue here

Comment (chasingSublimity) [reply CHANGELOG.md]: Like  hyperlink or just a parenthetical ref?

File: lib/claim.js [hunk 2]
```diff
@@ -57,6 +63,7 @@ request.variables = () => [
   { name: 'lead.trustedform_cert_url', type: 'string', required: true, description: 'TrustedForm Certificate URL' },
   { name: 'trustedform.scan_required_text', type: 'array', required: false, description: 'Required text to search snapshot for' },
   { name: 'trustedform.scan_forbidden_text', type: 'array', required: false, description: 'Forbidden text to search snapshot for' },
+  { name: 'trustedform.vendor', type: 'string', required: false, description: 'Lead vendor name sent to TrustedForm' },
```
Comment (cgrayson) [inline lib/claim.js]: Maybe include something in the description like, "defaults to lead source name"?

File: CHANGELOG.md (hunk 1 above)
Comment (cgrayson) [reply CHANGELOG.md]: I think I usually put a link, but either is good. The main value for me is being able to dig into any details from the issue when looking back later.

**Pull request merged**